contributions.html

```diff
@@ -70,44 +69,44 @@ <h1>Contributions</h1>
-  <li>Cash: <a href="https://www.hoyosconsulting.com" target="_blank">Hoyos Consulting</a></li>
+  <li>Switch Loan: <a href="https://www.cologix.com/" target="_blank">Cologix</a></li>
+  <li>Two PDUs: <a href="https://www.nexstardigital.com/publisher-solutions-group/" target="_blank">Lakana (Nexstar Broadcasting)</a></li>
+  <li>Cash: <a href="http://mnvoip.com/" target="_blank">Minnesota VoIP</a></li> <!-- Broken HTTPS. -->
+  <li>7500R Switch, 100G &amp; Two 10G Line Cards: <a href="https://www.arista.com/">Arista</a> arranged by <a href="https://usinternet.com/">US Internet (USI)</a></li>
+  <li>Cash: <a href="https://www.hoyosconsulting.com/" target="_blank">Hoyos Consulting</a></li>
   <li>Copper SFP for route server port: Jeremy Lumby</li>
   <li>Two 10G DAC cables: Jeremy Lumby</li>
-  <li>EX4200-24T Switch: <a href="http://compudyne.net" target="_blank">Compudyne</a></li>
+  <li>EX4200-24T Switch: <a href="https://www.compudyne.com/" target="_blank">Compudyne</a></li>
   <li>EX4500 4-port SFP+ Module (2nd module): Jeremy Lumby</li>
         <li>Cash: Marty Hannigan</li>
-  <li>Legal Fees: <a href="https://he.net" target="_blank">Hurricane Electric</a></li>
-  <li>Juniper Service Contract: <a href="https://wiktel.com" target="_blank">Wikstrom Telephone Company (Wiktel)</a></li>
+  <li>Legal Fees: <a href="https://he.net/" target="_blank">Hurricane Electric</a></li>
+  <li>Juniper Service Contract: <a href="https://wiktel.com/" target="_blank">Wikstrom Telephone Company (Wiktel)</a></li>
   <li>EX4500 4-port SFP+ Module (1st module): Jeremy Lumby</li>
   <li>EX4200 4-port SFP Module: Jeremy Lumby</li>
-  <li>Website Redesign: <a href="https://northstarfiber.com" target="_blank">North Star Fiber (was: MDU Ethernet)</a></li>
-  <li>Redundant Power Supplies: <a href="http://mnvoip.com" target="_blank">Minnesota VoIP</a></li>
-        <li>Redundant Power Supply for EX4500: <a href="https://nuvera.net">Nuvera Communications</a></li>
-  <li>Looking Glass Router: <a href="http://mnvoip.com" target="_blank">Minnesota VoIP</a></li>
-  <li>EX4200-24T Switch: <a href="https://juniper.net" target="_blank">Juniper Networks</a></li>
-  <li>Address Space: <a href="https://wins.net" target="_blank">Wisconsin Independent Networks</a></li>
-  <li>Cisco 4500 Ethernet Switch: <a href="https://www.iphouse.com" target="_blank">ipHouse</a></li>
-  <li>Cisco 3560 Ethernet Switch: <a href="https://www.cooperative-networks.com" target="_blank">Cooperative Network Services</a></li>
-  <li>Route Server Hardware: <a href="https://www.arcustech.com" target="_blank">Arcustech</a></li>
-  <li>State LLC Filing Fees: <a href="http://www.mankatonetworks.com" target="_blank">Mankato Networks</a></li>
-  <li>Logo Artwork: <a href="https://paulbunyan.net" target="_blank">Paul Bunyan Communications</a></li>
+  <li>Website Redesign: <a href="https://multiwav.com/" target="_blank">Arvig MultiWav (was: North Star Fiber)</a></li>
```

Comment:
This probably should stay as "was: MDU Ethernet".

Reply:
Richard,
Adding the trailing slash on the top-level web sites is a minor formatting thing. If those changes bother you then I will remove all of them and update my PR.

[Should You Have a Trailing Slash at the End of URLs?](https://searchfacts.com/url-trailing-slash/)

Leaving off the trailing slash for the LinkedIn page link causes a 302 redirect, per [Xenu link sleuth](http://home.snafu.de/tilman/xenulink.html).

I will update the Website redesign company former name.

Reply:
I would prefer to NOT add the trailing slashes to top-level names. For paths within a site, we should use whatever the canonical name is to avoid redirects, so I agree with the LinkedIn URL change.